css/main.css

```diff
@@ -591,3 +559,54 @@ li.navbar-link:hover a, li.navbar-sublink:hover {
 .navbar-link, .navbar-sublink {
     text-align: center;
 }
+
+.gallery {
+    display: flex;
+    justify-content: center;
+    margin: auto;
+    padding: 4vw;
+    flex-wrap: wrap;
+    width: 90vw;
+}
+
+.staff {
+    width: 100%;
+}
+
+.staff-container {
+    margin: 2vh;
+}
+
+.staff-container div.staff-image {
+    width: 22vh;
+    height: 22vh;
+    overflow: hidden;
+    background-size: cover;
+    padding: 0;
+    border-radius: 50%;
+}
+
+.staff-container:hover div.staff-image {
+    box-shadow:0 4px 12px 0 var(--color-translucent), 0 6px 20px 0 var(--color-translucent);
+}
+
+.staff-name {
+    bottom: 0px;
+    width: 100%;
+    text-align: center;
+    font-size: 1.1em;
+    padding-top: 2vh;
```

Comment:
Can you increase the spacing between the photo and the name? It looks a bit cramped at the moment.

Reply:
Done.